docs/source/examples/github.rst

```diff
@@ -176,3 +176,23 @@ This is the same as::
     me = g.me() # or me = g.user(your_user_name)
     if me.update(new_name, blog, company, bio=bio):
         print('Profile updated.')
+
+
+Logging in as an Application, or Application Installation
+---------------------------------------------------------
+
+To login as an application you only need the app_id and private key. 
+
+::
+    # start with anonymous GitHub object
+    gh = github3.GitHub()
+    
+    # login as application, and access installations
+	gh.login_as_app(private_key_pem=key_file_pem, app_id=app_identifier)
+    gh.app_installations()
+    
+	# OR login as application installation, and access permissable resources
+	gh.login_as_app_installation(private_key_pem=key_file_pem, app_id=app_identifier, installation_id=install_id)
+    access_token = gh.session.auth.token
```

Comment:
I'm not sure what is happening here

Reply:
Added additional comments. It seems several aspects of [Apps API](https://developer.github.com/v3/apps/) are not incorporated into the library, requiring a valid OAuth token (for the installation, not the application) .